### ISSUE TITLE
Output description isn't printing

### ISSUE THREAD

**Issue by kenk667** [hashicorp/terraform#21633]
provider.aws v2.13.0
### Terraform Version 

Terraform v0.12.1
```
...
```

### Terraform Configuration Files

```hcl
output "vpc_name" {
  value = var.vpc_name
  description = "This is the VPC name"
}

output "vpc_ip_range" {
  value = var.vpc_cidr
  description = "This is the IP CIDR range as a /16 subnet mask for your VPC"
}
```

### Expected Behavior

The outputs should show the description shown in the code above
### Actual Behavior

output doesn't show the description when apply has completed:
```
Apply complete! Resources: 20 added, 0 changed, 0 destroyed.

Outputs:

vpc_ip_range = 10.0.0.0
vpc_name = new_vpc
```
### Steps to Reproduce

create outputs with a description and terraform apply

**Comment by mildwonkey** [comment]
Hi @kenk667 !
This is expected behavior; the [`terraform output` command](https://www.terraform.io/docs/commands/output.html) only prints the value of outputs.

I can flag this as an enhancement request; it will help if you can share a bit about your use case.

**Comment by kenk667** [comment]
The use case is where I'd like to export the output as a file with a verbose printout of the descriptions. the assumption is that the code may be delivered and without some method of documentation it becomes hard to interpret what was done via the output with arbitrary output tags. An ideal solution would be an output flag that can save to local or remote location with the verbose description included

**Comment by AaronNHart** [comment]
Sorry for resurrecting a sleeping issue, but I would love to see this option as well.  Having the description field included, especially in the json output would make this a very nice and self contained method for documenting the current state of the configuration. This could be shared between teams by any number of convenient methods.

**Comment by mrowek** [comment]
just a flag to include description into output print would be great.
It is useful when people that are not familiar with terraform code need to check out output searching things for them. It will help make output much cleaner and readable for them

**Comment by nekodojo** [comment]
Agreed, Description is not really available to us, it needs to be in the JSON output at least.
